tests/kubic/repositories.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Configure Kubic repositories
+# Maintainer: Panagiotis Georgiadis <pgeorgiadis@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils "zypper_call";
+
+sub run {
+    if (get_var('MIRROR_HTTP')) {
```

Comment:
Please add comment about why we replace mirrors - I don't know and I think it's not obvious

Reply:
When I am cloning the job to test it locally, I have no access to the URLs.   e.g. `zypper ar -f http://openqa.opensuse.org/assets/repo/openSUSE-Tumbleweed-oss-i586-x86_64-Snapshot20181106` this always fails for me locally. Does it work for you?